docs/ref/forms/widgets.txt

```diff
@@ -183,6 +182,53 @@ In a nutshell, you will need to subclass the widget and either
 These methods involve somewhat advanced Python programming and are described in
 detail in the :doc:`Form Assets </topics/forms/media>` topic guide.
 
+Styling ``ChoiceWidget`` choices
+-------------------------------------
+
+.. versionadded:: 2.1
+
+Similarly to how you can style an entire wighet using the :attr:`Widget.attrs`
+attribute it is also possible to style individual choices for widgets inheriting
+from the ``ChoiceWidget`` class. To do so, you can pass a keyword argument
+``opt_attrs`` to the constructors of such widgets. This argument should be a
+dictionary containing HTML attributes to be set on all options generated by the
+widget.
+
+Besides constants, values may be given as callable types. If a callable option
+attribute is given it is called every time an option is created and is passed
+a keyword list with keys including ``value``, ``label`` and ``index``.
```

Comment:
"... passed a keyword list ..." isn't completely clear to me. It sounds like the function is receiving a single list argument. Personally I'd phrase this as "... passed keyword arguments including ..." to clarify this.